include/fdreadoutlibs/wibeth/tpg/ProcessStandardRSAVX2.hpp

```diff
@@ -196,7 +195,7 @@ process_window_standard_rs_avx2(ProcessingInfo<NREGISTERS>& info)
       hit_charge = _mm256_min_epi16(hit_charge, overflowMax);
 
 
-      //if(ireg==0){
+      if(ireg==0){
```

Comment:
I would prefer to be explicit about completing the comments.

```suggestion
      //if(ireg==0){
```